silkworm/silkrpc/concurrency/context_pool.hpp

```diff
@@ -130,7 +130,7 @@ class ContextPool {
     boost::asio::io_context& next_io_context();
 
   private:
-    static const std::size_t DEFAULT_POOL_STORAGE_SIZE = 0x400;
+    static const std::size_t DEFAULT_FILTER_STORAGE_SIZE = 1024;  // default filter storage size, ie max num for filters in storage
```

Comment:
Rename this constant accordingly to [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html)

Moreover, please move this constant in `filter_storage.hpp`

Reply:
Done